<?xml version="1.0"?>

<launch>

  <include file="$(find iturov_control)/launch/apm.launch" />

  <node name="joy_node" pkg="iturov_control" type="joy_node" output="screen" />
  <node name="PID_Params" pkg="iturov_control" type="server_pid.py" />
  <node name="Joy_Params" pkg="iturov_control" type="server_joystick.py" />

  <node name="joy_control_node" pkg="iturov_control" type="iturov_joy_pid_control.py" output="screen" />
  <!--
  <node name="dynamic_reconfigure_load" pkg="dynamic_reconfigure" type="dynparam" args="load
      camera_node_name $(find my_package)/params/camera.yaml"/>
  -->
</launch>